lib/components/player_sheet.dart

```diff
@@ -263,6 +267,55 @@ class PlayerSheetState extends State<PlayerSheet>
                     const Spacer(
                       flex: 6,
                     ),
+                    IconButton(
+                      constraints: const BoxConstraints.tightFor(
+                        width: 32,
+                        height: 32,
+                      ),
+                      padding: EdgeInsets.zero,
+                      onPressed: () async {
```

Comment:
If there is on another place the same on pressed functionality, then create a service and put there the functionality. To make the code DRY!